packages/block-library/src/template-part/edit/index.js

```diff
@@ -70,8 +70,12 @@ export default function TemplatePartEdit( {
 		return (
 			<TagName { ...blockProps }>
 				<Warning>
-					{ __(
-						'Template part has been deleted or is unavailable.'
+					{ sprintf(
+						/* translators: %s: template part slug */
```

Comment:
Nitpicking: Should "Template" be capitalized here?
```suggestion
						/* translators: %s: Template part slug */
```